libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1074,6 +1079,27 @@ void RC_Channel::do_aux_function(const aux_func_t ch_option, const AuxSwitchPos
         break;
     }
 
+    case AUX_FUNC::RETRACT_MOUNT: {
+#if HAL_MOUNT_ENABLED
```

Comment:
At some stage we need to canonicalise on whether the `#if` goes on the outside of the case statement.

If you don't put it on the inside like this then any time somebody tries the switch they'll get a message.

Reply:
I followed the copter pattern...do you want me to change it? if the mount is not built, wouldn't you want to have the failed init message if someone setup the switch? so the if define should be outside, right?